libs/designsystem/src/lib/components/list/list.component.html

```diff
@@ -79,7 +79,12 @@
 
 <ng-template #virtualItemsTemplate let-virtualScrollData>
   <div *uiScroll="let item of virtualScrollData; let i = index">
-    <div [class.item-wrapper]="!item.headingName" [class.section-header]="item.headingName">
+    <div
+      [class.item-wrapper]="!item.headingName"
```

Comment:
It seems like the item-wrapper class, and its styling, is what causes the 'bump' when scrolling a list with sections